app/code/Magento/ProductAlert/Block/Adminhtml/Catalog/Product/Edit/Tab/Alerts/Renderer/Stock.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Magento\ProductAlert\Block\Adminhtml\Catalog\Product\Edit\Tab\Alerts\Renderer;
+
+use Magento\Backend\Block\Widget\Grid\Column\Renderer\AbstractRenderer;
+use Magento\Framework\DataObject;
+use Magento\Framework\Message\ManagerInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Backend\Block\Context;
+use Magento\InventorySales\Model\StockResolver;
+use Magento\InventorySalesApi\Api\Data\SalesChannelInterface;
+
+class Stock extends AbstractRenderer
+{
+    /**
+     * @var StoreManagerInterface
+     */
+    protected $_storeManager;
+
+    /**
+     * @var StockResolver
+     */
+    protected $_stockResolver;
+
+    /**
+     * @var ManagerInterface
+     */
+    protected $_messageManager;
+
+    /**
+     * Stock constructor.
```

Comment:
Pls remove this extra comment
It is clear that is constructor

Reply:
ok, I will